run/run.go

```diff
@@ -62,11 +76,15 @@ func Run(o Options) error {
 	auth := createInnkeeperAuthentication(o)
 
 	// create data client
-	dataClient, err := createDataClient(o, auth)
+	dataClients, err := createDataClients(o, auth)
 	if err != nil {
 		return err
 	}
 
+	if len(dataClients) == 0 {
+		log.Println("warning: no route source specified")
```

Comment:
I thought of this, but I still prefer the warning only, because even if there is a dataclient, it can be empty, or all routes can get deleted while running, and the result is the same, so it feels more logical to me this way. Also, what if in a weird solution some script should generate the arguments for a process pipe where skipper is only a non-critical process among others.